src/nft/components/profile/list/PriceTextInput.tsx

```diff
@@ -121,9 +126,16 @@ export const PriceTextInput = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [listPrice])
 
-  const borderColor =
-    warningType !== WarningType.NONE && !focused
+  const percentBelowFloor = (1 - (listPrice ?? 0) / (asset.floorPrice ?? 0)) * 100
+
+  const warningColor =
+    showResolveIssues && !listPrice
       ? colors.red400
+      : warningType !== WarningType.NONE && !focused
+      ? (warningType === WarningType.BELOW_FLOOR && percentBelowFloor >= 20) ||
+        warningType === WarningType.ALREADY_LISTED
+        ? colors.red400
```

Comment:
is this the right color? it doesn't seem to match the other warning states:
<img width="836" alt="Screenshot 2023-02-04 at 1 38 26 PM" src="https://user-images.githubusercontent.com/9094792/216788658-a42be680-4b05-4ac1-a6df-36e5577fa793.png">

Reply:
Yes this was the color requested by design